lib/thor/actions/inject_into_file.rb

```diff
@@ -21,9 +21,14 @@ module Actions
     #     gems.split(" ").map{ |gem| "  config.gem :#{gem}" }.join("\n")
     #   end
     #
+    WARNINGS = { unchanged_no_flag: 'File unchanged! The supplied flag value not found!' }
+
     def insert_into_file(destination, *args, &block)
       data = block_given? ? block : args.shift
-      config = args.shift
+      config = args.shift || {}
+
+      config.merge!(after: /\z/) unless config.key?(:before) || config.key?(:after)
```

Comment:
```suggestion
      config[after] = /\z/ unless config.key?(:before) || config.key?(:after)
```

Reply:
Fair enough :)
I personally prefer `merge!` over `=` (both are methods anyway :)) but it does look simpler indeed 😃